crypto/src/commitments/kzg.rs

```diff
@@ -178,8 +178,7 @@ impl<const N: usize, F: IsPrimeField<RepresentativeType = UnsignedInteger<N>>, P
         y: &FieldElement<F>,
         p: &Polynomial<FieldElement<F>>,
     ) -> Self::Commitment {
-        let value = p.evaluate(x);
-        let mut poly_to_commit = p + Polynomial::new_monomial(-&value, 0);
+        let mut poly_to_commit = p + Polynomial::new_monomial(-y, 0);
```

Comment:
This should work now also
```suggestion
        let mut poly_to_commit = p -y;
```

Reply:
Done